x-pack/plugins/lens/common/expressions/datatable/utils.ts

```diff
@@ -16,8 +16,7 @@ export function isNumericFieldForDatatable(currentData: Datatable | undefined, a
   const column = currentData?.columns.find(
     (col) => col.id === accessor || getOriginalId(col.id) === accessor
   );
-  // min and max aggs are reporting as number but are actually dates - work around this by checking for the date formatter until this is fixed at the source
```

Comment:
now that the min and max aggs were fixed, I think we can safely remove it.

Reply:
I think we could also remove the code in line 24, but I am leaving it just in case if there's something I miss!